templates/trust/contracts/TrustTemplate.sol

```diff
@@ -108,15 +108,20 @@ contract TrustTemplate is BaseTemplate {
     }
 
     function _setupApps(Kernel _dao, address[] _beneficiaryKeys, address[] _heirs, uint256[] _heirsStakes, uint256 _blockedHeirsSupply) internal {
+
+        // Install apps
         ACL acl = ACL(_dao.acl());
         Vault vault = _installVaultApp(_dao);
         Agent agent = _installNonDefaultAgentApp(_dao);
         Finance finance = _installFinanceApp(_dao, vault, 30 days);
         (Voting holdVoting, Voting heirsVoting, TokenManager holdTokenManager, TokenManager heirsTokenManager) = _installTokenApps(_dao);
 
-        _mintHoldTokens(acl, holdTokenManager, _beneficiaryKeys);
-        _mintHeirsTokens(acl, heirsTokenManager, _heirs, _heirsStakes, _blockedHeirsSupply);
+        // Mint tokens
+        _mintTokens(acl, holdTokenManager, _beneficiaryKeys, 1e18);
+        _mintTokens(acl, heirsTokenManager, _heirs, _heirsStakes);
+        _mintTokens(acl, heirsTokenManager, address(0), _blockedHeirsSupply);
```

Comment:
Note that this is fairly gas inefficient, since we assigning and re-assigning permissions on the `heirsTokenManager` here.

Although uglier, we could stuff this burned minting into the previous call instead.

Reply:
I _think_ I see your point. However, `heirsTokenManager` is not available in the previous call, se we can't move the mintint to the zero address to there. It only be comes available in this function, a few lines back at L116.

Reply:
Ah yes, I meant the previous call to `_mintTokens()`, directly above. We could add the `address(0), _blockedHeirsSupply` into a new array copied from `_heirsStakes` (although admittedly this is getting complex).

Reply:
Yep. That would indeed be more efficient, but quite obscure, don't you think? I suggest that we drop the conversation here (since its a closed PR) and create a new issue to address this inefficiency.

Reply:
Yes, agreed. Especially on a template that already requires quite a lot of gas, and may not be used all the time, I think this is acceptable for now (less so for something like the company template 😄 ).